Header header
int8[] channels
float64[] voltages

